neurodamus/node.py

```diff
@@ -933,7 +937,7 @@ def _report_build_params(self, rep_name, rep_conf, target, pop_offsets_alias_pop
             end_time,
             SimConfig.output_root,
             None,
-            Nd.String(rep_conf["Scaling"]) if "Scaling" in rep_conf else None,
+            rep_conf["Scaling"] if "Scaling" in rep_conf else None,
```

Comment:
```suggestion
            rep_conf.get("Scaling"),
```
Assuming this is just a `dict`?